docs/pages/guides/app-icons.md

```diff
@@ -24,7 +24,7 @@ Create an app icon and splash image with the [Figma template](https://www.figma.
 
 - The icon you use for iOS should follow the [Apple Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/icons-and-images/app-icon/) for iOS Icons.
 - Use a png file.
-- 1024x1024 is a good size. The Expo [build service](../distribution/building-standalone-apps.md) will generate the other sizes for you. The largest size it generates is 1024x1024.
+- 1024x1024 is a good size. [EAS Build](/build/setup.md) will generate the other sizes for you. The largest size it generates is 1024x1024.
```

Comment:
maybe makes more sense to point to prebuild here, or just in general that this is only applies to managed apps